### DISCUSSION TITLE
Spark.Sql C# Functions

### DISCUSSION THREAD

**Discussion by rohitpitre** [dotnet/spark#1012] [Q&A]
Hi All, 

Has anyone used the Spark.Sql C# function in Synapse notebooks ? I am trying from last 4-5 days and unable to find sufficient info/examples on the functions like IsNull, Equals, EqualsTo. Any reference material would be of great help, please share.
**https://docs.microsoft.com/en-us/dotnet/api/microsoft.spark.sql.functions?view=spark-dotnet**

Regards,
Rohit

**Comment by MikeRys** [comment]
Hi Rohit.

We do have sample scripts that use Spark.Sql in .NET notebooks for example here: Synapse/Notebooks/Spark.NET C#/Tweets at main * Azure-Samples/Synapse * GitHub<https://github.com/Azure-Samples/Synapse/tree/main/Notebooks/Spark.NET%20C%23/Tweets>

Are you referring to the SQL expressions that Spark allows you to use? These expressions are the same as the SQL expressions that you can use with SparkSQL in any other language embedding (e.g., Scala, Python etc).

Best regards
Michael


From: rohitpitre ***@***.***>
Sent: Friday, January 7, 2022 3:18 AM
To: dotnet/spark ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [dotnet/spark] Spark.Sql C# Functions (Discussion #1012)


Hi All,

Has anyone used the Spark.Sql C# function in Synapse notebooks ? I am trying from last 4-5 days and unable to find sufficient info/examples on the functions like IsNull, Equals, EqualsTo. Any reference material would be of great help, please share.
https://docs.microsoft.com/en-us/dotnet/api/microsoft.spark.sql.functions?view=spark-dotnet<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fdotnet%2Fapi%2Fmicrosoft.spark.sql.functions%3Fview%3Dspark-dotnet&data=04%7C01%7Cmrys%40microsoft.com%7C6e301bcdcd53454787ad08d9d1cf66eb%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637771510991790890%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=irSWa3J4r9SRwFr9cPWLVbJbHw2krNBU91WcifI8csA%3D&reserved=0>

Regards,
Rohit

-
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdotnet%2Fspark%2Fdiscussions%2F1012&data=04%7C01%7Cmrys%40microsoft.com%7C6e301bcdcd53454787ad08d9d1cf66eb%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637771510991790890%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=FsnAIhppRvgymP8XDJV2uk1zqar9jQVqu6ZFPsoNGF8%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FACZXGJET5MCNURRY4KOKL33UU3DXRANCNFSM5LOSFPZQ&data=04%7C01%7Cmrys%40microsoft.com%7C6e301bcdcd53454787ad08d9d1cf66eb%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637771510991790890%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=vrQOYa5JEWzOTPpBDEDpQtcsmbGHC0NRgcj7Wf813s4%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cmrys%40microsoft.com%7C6e301bcdcd53454787ad08d9d1cf66eb%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637771510991790890%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=eVVRo7jqk%2BBaYezZDEGePOF3FaPkoxp6II8plYkzDGo%3D&reserved=0> or Android<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cmrys%40microsoft.com%7C6e301bcdcd53454787ad08d9d1cf66eb%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637771510991790890%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=oMm4Iex0Y%2ByM4OVbqFowFf9buZcOoz1lC%2FyhcJ24jCE%3D&reserved=0>.

**Comment by rohitpitre** [comment]
Thanks Michael. I will have a look at the examples and get back to you in
case I have any questions.

Regards,
Rohit
